frontend/src/components/WofPopper.vue

```diff
@@ -0,0 +1,25 @@
+<template>
+    <div class="popper"> <slot></slot> </div>
```

Comment:
Make sure that everything we put in this component will be displayed in the center of site.